<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="PendingActivation">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>200</number>
        </property>
        <property name="topMargin">
         <number>60</number>
        </property>
        <property name="rightMargin">
         <number>200</number>
        </property>
        <property name="bottomMargin">
         <number>60</number>
        </property>
        <property name="spacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_ProductSelect">
          <item>
           <widget class="QLabel" name="product_emoji">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>src/PendingActivation.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_info">
            <property name="text">
             <string>Before entering the user interface, 
you need to enter the activation ID and 
select the product</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_ActiveID">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_PendingIdentityID">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_SelectProduct">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Select Product</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_PendingIdentityID">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Serial Number</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QLabel" name="label_Author">
              <property name="text">
               <string>Designed by github.com@Sodeug</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_SelectProduct">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Select Product</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_LastUpdate">
              <property name="text">
               <string>Last Update Date: 2025/09/06</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_SelectProduct">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>InteractiveHtml</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_ActiveID">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Active ID</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="InteractiveHtmlSetup">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>160</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>160</number>
          </property>
          <property name="bottomMargin">
           <number>60</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_setup">
            <item>
             <widget class="QLabel" name="label_Emoji">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>src/welcom.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_status">
              <property name="text">
               <string>Status: Ready...</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="spacing">
               <number>10</number>
              </property>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_InteractiveHtmlBomID">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_CadenceDirectory">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_PatchID">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Patch ID</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_CadenceDirectory">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Cadence Directory</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_InteractiveHtmlBomID">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>InteractiveHtmlBom ID</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_exportJsonID">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_exportJsonID">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>exportJson ID</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_PatchID">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_CheckEnvironment">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>false</bold>
                <underline>false</underline>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
              <property name="text">
               <string extracomment="检测环境">Check Environment</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+S</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_InstallPatch">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Install</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="CadenceInteractiveHtmlAbout">
              <property name="text">
               <string>UI
github@SoDebug
Credits
github@openscopeproject\InteractiveHtmlBom
github@juulsA\exportJson</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
